sdk/python/tests/compiler/testdata/loop_static.py

```diff
@@ -17,20 +17,20 @@
 
 @dsl.pipeline(name='my-loop-pipeline')
 def pipeline(my_pipe_param='10'):
-    loop_args = [{'A_a': 1, 'B_b': 2}, {'A_a': 10, 'B_b': 20}]
```

Comment:
If the custom task cannot support looping over a dictionary, should we keep the old implementation for flatten list of dictionary arguments?

Reply:
The dictionary supporting is on the way, I am doing it right now, expect it will be ready soon. To support it, I need to update the reconciler of pipelineloop, since this is not a dataStage case, that's why I didn't impletmented it.